App/Screens/About/About.tsx

```diff
@@ -142,18 +204,6 @@ export function About (props: AboutProps) {
         </Text>
       </View>
 
-      <View style={styles.distance}>
```

Comment:
Ah, sorry, forgot about this one. Another merge error: This big chunk has been forgotten.